burrow/check.py

```diff
@@ -0,0 +1,86 @@
+# (C) Datadog, Inc. 2010-2017
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+# 3rd party
+import requests
+
+# project
+from checks import AgentCheck
+
+EVENT_TYPE = SOURCE_TYPE_NAME = 'burrow'
+
+
+class Burrow(AgentCheck):
+
+    LIST_CLUSTERS = '/v2/kafka'
+    CLUSTER_DETAILS = '/v2/kafka/{cluster}'
+    LIST_CONSUMERS = '/v2/kafka/{cluster}/consumer'
+    CONSUMER_LAG = '/v2/kafka/{cluster}/consumer/{consumer}/lag'
+
+    @staticmethod
+    def _check_for_error(req):
+        if req.status_code != 200:
+            raise Exception("Got {} error from burrow http endpoint".format(req.status_code))
+
+        res = req.json()
+        if res['error']:
+            raise Exception("Got error from burrow http endpoint: {}".format(res['message']))
+
+        return res
+
+    def __init__(self, name, init_config, agentConfig, instances=None):
+        AgentCheck.__init__(self, name, init_config, agentConfig, instances)
+
+    def check(self, instance):
+        if 'status_url' not in instance:
+            raise Exception("Missing 'status_url' in Burrow config")
+
+        url = instance['status_url']
+        connect_timeout = int(instance.get('connect_timeout', 5))
+        receive_timeout = int(instance.get('receive_timeout', 15))
+
+        optional_tags = instance.get('tags', [])
+
+        r = requests.get(url=url+Burrow.LIST_CLUSTERS,
```

Comment:
could you replace all references of `Burrow` by `self`? More consistent with the rest of the integrations' codebase